src/core/selfcare.tf

```diff
@@ -222,7 +222,7 @@ module "appservice_selfcare_be" {
     FAILURE_URL         = "https://${local.selfcare_io.frontend_hostname}/500.html"
     SELFCARE_LOGIN_URL  = "https://${local.selfcare.hostname}/auth/login"
     SELFCARE_IDP_ISSUER = "https://${local.selfcare.hostname}"
-    SELFCARE_JWKS_URL   = data.http.selfcare_well_known_jwks_json.url
+    SELFCARE_JWKS_URL   = "https://${local.selfcare.hostname}/.well-known/jwks.json"
```

Comment:
@balanza can we safe delete `data "http" "selfcare_well_known_jwks_json"`?
Now is used only to reference jwks url without reference the json in the response.

I found this because today I notice that we a problem to get response from `https://uat.selfcare.pagopa.it/.well-known/jwks.json`. Probably an Azure CDN issue.

https://dev.azure.com/pagopaspa/io-platform-iac-projects/_build/results?buildId=45685&view=logs&j=1ed085e6-09af-57e4-ce2a-d888424f4354&s=d3a93255-6532-5878-6486-d28b8ffe050e&t=ccd61e70-94ec-56f6-7677-8c183acbdbd3&l=743

Anyway if not used less is better

Reply:
yes, it's just a refactor you can proceeed